Classes/Driver/Version5/Query/FilteredQuery.php

```diff
@@ -61,6 +61,7 @@ public function fulltext(string $searchWord, array $options = []): void
     {
         $this->appendAtPath('query.bool.must', [
             'query_string' => array_merge($options, [
+//            'multi_match' => array_merge($options, [
```

Comment:
Hey @gerhard-boden - seems something went wrong here :)

Reply:
Oh yes, was testing if my change broke the test or if the tests have been broken all along. But obviously it's my change 🙈 (Didn't want to install elasticsearch locally for just running the test)

Will revert to my change and try to figure out why I'm breaking the tests.